### PR TITLE
Add license MIT

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 netx.js developers
```
Comment (jamo) [inline LICENSE.md]: typo in the word `next`

Comment (hngiang) [inline LICENSE.md]: Nice find @jamo , i fix that.

File: README.md [hunk 2]
```diff
@@ -378,3 +378,7 @@ The following issues are currently being explored and input from the community i
 - Tony Kovanen ([@tonykovanen](https://twitter.com/tonykovanen)) – ▲ZEIT
 - Guillermo Rauch ([@rauchg](https://twitter.com/rauchg)) – ▲ZEIT
 - Dan Zajdband ([@impronunciable](https://twitter.com/impronunciable)) – Knight-Mozilla / Coral Project
+
+## License
+
+MIT
```
Comment (iamstarkov) [inline README.md]: LICENSE file is explicit and git hosting agnostic

Comment (flybayer) [inline README.md]: @iamstarkov yeah, I agree the LICENSE file should be in the repo, but I think we don't need it added to the README.

BTW, the license is also declared in [package.json](https://github.com/zeit/next.js/blob/master/package.json#L6)

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 next.js developers
```
Comment (nkzawa) [inline LICENSE]: I think copyright should be `Zeit, Inc.`.

Comment (hngiang) [inline LICENSE]: I change copyright to Zeit, Inc.

File: README.md [hunk 4]
```diff
@@ -378,3 +378,4 @@ The following issues are currently being explored and input from the community i
 - Tony Kovanen ([@tonykovanen](https://twitter.com/tonykovanen)) – ▲ZEIT
 - Guillermo Rauch ([@rauchg](https://twitter.com/rauchg)) – ▲ZEIT
 - Dan Zajdband ([@impronunciable](https://twitter.com/impronunciable)) – Knight-Mozilla / Coral Project
+
```
Comment (rauchg) [inline README.md]: why this `\n`?

**Review by rauchg — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (hngiang) [inline README.md]: Default `README.md` has `\n`

Comment (nkzawa) [inline README.md]: Could you remove it since it's not related to license ?
And please squash all commits to one.

**Comment by rauchg** [comment]
Thank you!

**Pull request merged**